src/common/util.tsx

```diff
@@ -80,3 +82,21 @@ export const getLocalFocusData = (): string[] => {
   }
   return JSON.parse(localData) as string[];
 };
+
+export const getLocalQuestListDepthData = (): number => {
+  const localQuestListDepth = localStorage.getItem("questDepth");
+  if (localQuestListDepth) {
+    return +localQuestListDepth;
```

Comment:
What does this + do? I can't tell if I've never seen this syntax or if it's a typo.

Reply:
oh I see it coerces a string into a number